src/components/Projects/WBS/WBSDetail/WBSTasks.jsx

```diff
@@ -201,6 +204,10 @@ const WBSTasks = props => {
   });
   const filteredTasks = filterTasks(LoadTasks, filterState);
 
+  useEffect(() => {
```

Comment:
same cleanup questions/reminder for this useEffect() as well, seems that its only used for logging